container/lxd/client_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package lxd_test
+
+import (
+	"github.com/golang/mock/gomock"
+	gc "gopkg.in/check.v1"
+
+	"github.com/juju/juju/container/lxd"
+	lxdtesting "github.com/juju/juju/container/lxd/testing"
+	coretesting "github.com/juju/juju/testing"
+	"github.com/lxc/lxd/shared/api"
```

Comment:
This import should be in the previous block.

Reply:
Acknowledged.